docs/how-tos/howto-setup-hashicorp-vault.md

```diff
@@ -0,0 +1,99 @@
+
+The node operator that's starting the Polygon SDK can now specify which secrets manager they want to use, and as soon 
+as the correct secrets manager is instantiated, the modules deal with the secrets through the mentioned interface - 
+without caring if the secrets are stored on disk or on a server.
+
+This article details the necessary steps to get the Polygon SDK up and running with a [Hashicorp Vault](https://www.vaultproject.io/) server.
+
+:::info previous guides
+It is **highly recommended** that before going through this article, articles on [**How to Set Up IBFT Locally**](/docs/how-tos/howto-setup-ibft/howto-set-ibft-locally) 
+and [**How to Set Up IBFT on the Cloud**](/docs/how-tos/howto-setup-ibft/howto-set-ibft-on-the-cloud) are read.
+:::
+
+
+## Prerequisites
+
+This article assumes that a functioning instance of the Hashicorp Vault server **is already set up**.
+
+Additionally, it is required that the Hashicorp Vault server being used for the Polygon SDK should have **enabled KV storage**.
+
+Required information before continuing:
+* **The server URL** (the API URL of the Hashicorp Vault server)
+* **Token** (access token used for access to the KV storage engine)
+
+## Step 1 - Generate the secrets manager configuration
+
+In order for the Polygon SDK to be able to seamlessly communicate with the Vault server, it needs to parse an already
+generated config file, which contains all the necessary information for secret storage on Vault.
+
+To generate the configuration, run the following command:
+
+```bash
+go run main.go secrets-manager generate --dir <PATH> --token <TOKEN> --server-url <SERVER_URL> --name <NODE_NAME>
+```
+
+Parameters present:
+* `PATH` is the path to which the configuration file should be exported to. Default `./secretsManagerConfig.json`
+* `TOKEN` is the access token previously mentioned in the [prerequisites section](/docs/how-tos/howto-setup-hashicorp-vault#prerequsites)
+* `SERVER_URL` is the url of the API for the Vault server, also mentioned in the [prerequisites section](/docs/how-tos/howto-setup-hashicorp-vault#prerequsites)
+* `NODE_NAME` is the name of the current node for which the Vault configuration is being set up as. It can be an arbitrary value. Default `polygon-sdk-node`
+
+:::caution Node names
+Be careful when specifying node names.
+
+The Polygon SDK uses the specified node name to keep track of the secrets it generates and uses on the Vault instance.
+Specifying an existing node name can have consequences of data being overwritten on the Vault server.
+
+Secrets are stored on the following base path: `secrets/node_name`
+:::
+
+## Step 2 - Initialize IBFT using the configuration
+
+Now that the configuration file is present, we can initialize required IBFT data (secret keys) with the configuration 
+file set up in step 1, using the `--secrets-config`:
+
+```bash
+go run main.go secrets init --secrets-config <PATH>
```

Comment:
This is correct one?

```shell
go run main.go secrets init --config <PATH>
```

Reply:
Yep, another good catch 💯 

Fixed 👍 

4dc3836